src/web/src/components/LandingButtons.tsx

```diff
@@ -0,0 +1,70 @@
+import Link from 'next/link';
+import { makeStyles } from '@material-ui/core';
+import Button from '@material-ui/core/Button';
+import useAuth from '../hooks/use-auth';
+
+const useStyles = makeStyles((theme) => ({
+  buttonsContainer: {
+    position: 'absolute',
+    color: 'white',
+    top: '20px',
+    width: '250px',
+    right: '5%',
+    display: 'flex',
+    justifyContent: 'space-around',
+    '& > .MuiButton-root': {
+      color: 'white',
+      border: '1.5px solid white',
+      '&:hover': {
+        color: '#9ABDFF',
+      },
+    },
+    [theme.breakpoints.down(490)]: {
+      width: '100%',
+      right: 0,
+    },
+  },
+}));
+
+const LandingButtons = () => {
+  const classes = useStyles();
+  const { login } = useAuth();
+
+  return (
+    <div className={classes.buttonsContainer}>
+      <Link href="/about" passHref>
+        <Button
+          style={{
+            border: 'none',
+            padding: 0,
+            fontSize: '1.2rem',
+          }}
+          variant="outlined"
+        >
+          About us
+        </Button>
+      </Link>
+      <Button
+        style={{
+          border: 'none',
+          padding: 0,
+          fontSize: '1.2rem',
+        }}
+        onClick={() => login()}
+        variant="outlined"
+      >
+        Sign in
+      </Button>
+      <Button
+        style={{
+          fontSize: '1.3rem',
+        }}
+        variant="outlined"
+      >
+        Sign up
+      </Button>
+    </div>
```

Comment:
Can we reuse code here? The 3 buttons are almost identical, so let's do something similar to [this](https://github.com/Seneca-CDOT/telescope/blob/master/src/web/src/components/NavBar/index.tsx)

Reply:
Also, we should add logic here to render the Sign in and Sign up buttons **or** the user's name if they're logged in.

Reply:
You are right about the `sign in` and `sign up`. I made this change. But I decided to keep the buttons the way they are. It's working. If you want, we can render the buttons using `map` in a follow-up.

Reply:
Up to you, but I think it's a waste of time. The buttons thing is a very small change, perfectly doable in a few lines, and it's related to what you're adding in this PR. Filing a new issue, picking someone to work on it, making the changes, submitting a new PR, reviewing that PR... Wouldn't it be better to just add do it here and save all those extra steps?